src/query_parser/v2/parser.y

```diff
@@ -1174,7 +1182,14 @@ exclusive_param_num(A) ::= LP num(B). {
 
 exclusive_param_num(A) ::= LP ATTRIBUTE(B). {
     A = B;
+    A.sign = 1;
```

Comment:
Can also set the default value of `A.sign` to 1. I think it's not initialized now

Reply:
OK. Done.